database/templates/database/species_detail.html

```diff
@@ -4,42 +4,48 @@
 <h1>Detail</h1>
 
     <h2>Names</h2>
-    <ul>
+    <ul class="list-group">
         {% for name in names %}
-            <li>{{ name }}</li>
+            <li class="list-group-item">{{ name }}</li>
         {% endfor %}
     </ul>
     <hr>
     <h2>Adjacency Lists</h2>
-    <ul>
+    <ul class="list-group">
         {% for adjlist in adjlists %}
-            <li>{{ adjlist }}</li>
+            <li class="list-group-item">{{ adjlist }}</li>
         {% endfor %}
     </ul>
     <hr>
     <h2>SMILEs</h2>
-    <ul>
+    <ul class="list-group">
         {% for smile in smiles %}
-            <li>{{ smile }}</li>
+            <li class="list-group-item">{{ smile }}</li>
         {% endfor %}
     </ul>
     <hr>
     <h2>Isomer Inchis</h2>
-    <ul>
+    <ul class="list-group">
         {% for isomer_inchi in isomer_inchis %}
-            <li>{{ isomer_inchi }}</li>
+            <li class="list-group-item">{{ isomer_inchi }}</li>
+        {% endfor %}
+    </ul>
+    <h2>Structures</h2>
+    <ul class="list-group">
+        {% for structure in structures %}
+        <li class="list-group-item"><img src="{% url 'draw-structure' structure.pk %}"/></li>
```

Comment:
These should correspond one-to-one with the Adjacency Lists listed above. Is that true, and conveyed visually or syntactically?

Reply:
now in a table. thanks